packages/teams-js/src/public/app.ts

```diff
@@ -550,6 +554,11 @@ export namespace app {
         Handlers.initializeHandlers();
         GlobalVars.initializePromise = initializeCommunication(validMessageOrigins).then(
           ({ context, clientType, runtimeConfig, clientSupportedSDKVersion = defaultSDKVersionForCompatCheck }) => {
+            initializeHelperLogger('context: %s', context);
```

Comment:
This should already be printed as part of the communication layer logging?

Reply:
You are correct, communication.handleParentMessage. Was a little hard to correlate because the message name wasn't being logged